thorlcr/graph/thgraphslave.hpp

```diff
@@ -252,6 +253,7 @@ class graphslave_decl CSlaveActivity : public CActivityBase, public CEdgeProgres
     void stopInput(unsigned index, const char *extra=NULL);
     void stopAllInputs();
     virtual void serializeStats(MemoryBuffer &mb);
+    virtual void serializeActivityStats(MemoryBuffer &mb);
```

Comment:
can be const method.

Reply:
It would be nice to but serializeActivityStats is used by serializeStats and serializeStats is not const.  I can't make serializeStats const method because a derived class overrides serializeStats and that implementation updates some class variables.

Reply:
>It would be nice to but serializeActivityStats is used by serializeStats and serializeStats is not const

not sure I follow, just because calling method isn't const, doesn't mean something it calls can't be...

Reply:
I'm not sure what I was thinking.  Probably, confused with a compiiler error message elsewhere. Fixing